fs/fshttp/http.go

```diff
@@ -72,16 +72,20 @@ func NewTransportCustom(ctx context.Context, customize func(*http.Transport)) ht
 		t.TLSClientConfig.Certificates = []tls.Certificate{cert}
 	}
 
-	// Load CA cert
-	if ci.CaCert != "" {
-		caCert, err := os.ReadFile(ci.CaCert)
-		if err != nil {
-			log.Fatalf("Failed to read --ca-cert: %v", err)
-		}
+	// Load CA certs
+	if ci.CaCert != nil && len(ci.CaCert) != 0 {
```

Comment:
You only need `len(ci.CaCert) != 0` as `len` on a `nil` slice is defined to be 0.